source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -193,6 +193,22 @@ As an alternative to `drush sql-sync` you can use `drush sql-dump` instead.
 
 Commands that alter site code, such as pm-download (dl), will only work on a Dev environment that has been set to [SFTP mode](/docs/articles/sites/code/developing-directly-with-sftp-mode/) from the Pantheon Dashboard.
 
+## Add Custom Drush Commands
+
+While we have the full spectrum of drush core already available for your use, you may want to add a command that you regularly use, for instance, [drush search and replace (sar)](https://www.drupal.org/project/sar). 
+  
+1. Put the site in Git mode.
+2. Clone locally.
+3. Create a “drush” folder in the Drupal root.
+4. Add the “sar” drush command to the “drush” folder.
+5. Commit drush/sar.
+6. Push your code up to master. 
+7. Deploy to Test and Live.
+8. Download your Pantheon [drush aliases](https://pantheon.io/blog/drush-aliases-available).
+9. Clear your drush cache on each environment. (example: “drush @pantheon.<site-name>.devcc drush”)
+
+If you have successfully set up [Terminus](https://pantheon.io/docs/articles/local/cli/), the Pantheon CLI, you can get your drush aliases by using “terminus sites aliases”. At this point, you are able to start using the drush command you added.  Drush 5 is the default version for newly created sites on Pantheon.  However, to have strict control over your drush site aliases, see [Fix Up Drush Site Aliases with a Policy File](https://pantheon.io/blog/fix-drush-site-aliases-policy-file).

```

Comment:
Let's apply the same principle here - get the "how to" out of the blog and add it to this doc.